applications/CoSimulationApplication/tests/test_coupling_operations.py

```diff
@@ -0,0 +1,164 @@
+from __future__ import print_function, absolute_import, division  # makes these scripts backward compatible with python 2.6 and 2.7
+
+import KratosMultiphysics as KM
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+from KratosMultiphysics.CoSimulationApplication.coupling_interface_data import CouplingInterfaceData
+from KratosMultiphysics.CoSimulationApplication.factories import coupling_operation_factory
+
+from KratosMultiphysics.CoSimulationApplication.co_simulation_tools import UsingPyKratos
+using_pykratos = UsingPyKratos()
+
+from math import sqrt, pi
+
+class SolverWrapper(object):
```

Comment:
can't we use a the dummy solver wrapper from the helpers ?

Reply:
hm in principle yes, but then there is a lot more overhead bcs this also needs settings and other input